tests/test_api.py

```diff
@@ -88,3 +88,12 @@ def test_delete_all_channels(self, app):
         app.api.delete_all_channels()
         channels = app.api.get_channels()
         assert channels == []
+
+    def test_delete_channel(self, app, random_name):
+        name = random_name()
+        channel_id = app.api.create_channel(name)
+        status_code = app.api.delete_channel(channel_id)
+        assert isinstance(status_code, int)
```

Comment:
I think we need assert status_code value too, because for some reasons deletion may not work.